src/modules/weather.js

```diff
@@ -0,0 +1,102 @@
+const weatherModule = () => {
+  const locationInput = document.getElementById('location-input');
+  const submitButton = document.getElementById('submit-weather-form');
+  const nameElemenet = document.getElementById('name');
+  const tempElement = document.getElementById('temp');
+  const weatherElement = document.getElementById('weather');
+  const humidityElement = document.getElementById('humidity');
+  const dataContainer = document.getElementById('data-container');
+  const loader = document.getElementById('loader-container');
+  const tempToggle = document.getElementById('temp-toggle');
+
+
+  const fetchWeather = async (location, unit = false) => {
+    const unitValue = unit === true ? 'metric' : 'imperial';
+    const response = await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${location}&units=${unitValue}&appid=4bd4b9ab3b3c5a1d664f8bc0eecfaf36`, { mode: 'cors' });
+    loader.style.display = 'flex';
+    const weatherData = await response.json();
+    return weatherData;
+  };
+
+
+  const parseWeatherJSON = (json) => {
+    const { temp } = json.main;
+    const { name } = json;
+    const weather = json.weather[0].main;
+    const { humidity } = json.main;
+
+    return {
+      name, temp, weather, humidity,
+    };
+  };
+
+  async function getWeatherImage(weather) {
+    const response = await fetch(`https://api.pexels.com/v1/search?query=${weather}+weather&size=medium`, {
+      headers: {
+        Authorization: '563492ad6f91700001000001abc669dc2c4e46d28693596b6f0a47bf',
+      },
+    }, {
+      mode: 'cors',
+    });
+
+    const image = await response.json();
+    return image.photos[0].src.original;
+  }
```

Comment:
I think it would be nice to make use of the arrow function syntax here as well. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))